mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po

```diff
@@ -3438,3 +3476,74 @@ msgstr ""
 
 msgid "spyder_flower_sandy_notweesher"
 msgstr "Oh, no importa. Veo que no tienes un Tweesher."
+
+msgid "spyder_route2_roddick1"
+msgstr "Cuando fui emboscado en la duna por SPIGHTER estuve muy asustado."
+
+msgid "spyder_route2_graf2"
+msgstr "Me sirvieron."
+
+msgid "spyder_wayfarer1_bismuth2"
+msgstr "¡Gorrón!"
+
+msgid "spyder_route2_roddick2"
+msgstr "Oye, ¡no parecías arrastrado en absoluto!"
+
+msgid "menu_down_key"
+msgstr "Tecla Bajar"
```

Comment:
Nobody says that in Spanish. It could be "Flecha Abajo" or better, simply "Abajo" (which even makes sense for a controller).
```suggestion
msgstr "Abajo"
```